apps/chatter/serializers.py

```diff
@@ -15,9 +24,10 @@ class Meta:
         extra_kwargs = {'author': {'read_only': True}}
 
 class MessageSerializer(serializers.Serializer):
-    command = serializers.ChoiceField(choices=['INIT_CHAT', 'FETCH_ENTRIES', 'NEW_ENTRY', 'ENTRIES'])
+    command = serializers.ChoiceField(choices=['INIT_CHAT', 'FETCH_ENTRIES', 'NEW_ENTRY'])
```

Comment:
Wonder if this should reference the constants on the `ChatConsumer` class?

```
choices=[ChatConsumer.INIT_CHAT, ChatConsumer.FETCH_ENTRIES, ...]
```

Reply:
Oh yeah!